tests/e2e/seed.spec.ts

```diff
@@ -13,9 +20,11 @@ test("seed metadata", async ({ page }) => {
     "alt",
     "🚀",
   );
-  await expect(page.locator("text=hyb6i8…sg6nn4")).toBeVisible();
-  await expect(page.locator("text=8777")).toBeVisible();
-  await expect(page.locator("text=0.2.0")).toBeVisible();
+  await expect(
+    page.locator(`text=${remote.substring(0, 6)}…${remote.substring(42)}`),
+  ).toBeVisible();
+  await expect(page.locator(`text=${seedPort}`)).toBeVisible();
+  await expect(page.locator(`text=${seedVersion}`)).toBeVisible();
```

Comment:
instead of two substring we can define another const `ridFormatted` which would be `"zKtT7D…19WzjT"`